go.mod

```diff
@@ -72,13 +72,13 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dimchansky/utfbom v1.1.1 // indirect
 	github.com/docker/cli v23.0.5+incompatible // indirect
-	github.com/docker/distribution v2.8.2-beta.1+incompatible // indirect
+	github.com/docker/distribution v2.8.2+incompatible // indirect
```

Comment:
Same as [PR3863](https://github.com/open-horizon/anax/pull/3863), please change line 139 from:
```
github.com/docker/distribution => github.com/docker/distribution v2.8.2-beta.1+incompatible
```
to
```
github.com/docker/distribution => github.com/docker/distribution v2.8.2+incompatible
```
Then re-generate `go.sum`